qcfractal/interface/client.py

```diff
@@ -273,12 +276,12 @@ def get_results(self, **kwargs):
         else:
             return r.json()["data"]
 
-    def get_procedures(self, procedure_id, **kwargs):
+    def get_procedures(self, procedure_id, return_objects=True):
```

Comment:
Will there ever be a reason to provide other KWargs?

Reply:
I don't think there will be a reason for general `**kwargs` here, but there may be additional `kwargs`.